website_geoengine/readme/CONTIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Stéphane Brunner <stephane.brunner@camptocamp.com>  
```

Comment:
Typo in this file name, that's why you don't appear in the readme file as contributor.

Reply:
Corrected !